README.adoc

```diff
@@ -273,7 +273,9 @@ make cypress-gui
 
 Note that `make cypress-gui` runs the Cypress GUI that allows you to pick which individual tests to run. To run the entire test suite in headless mode, use the make target `cypress-run` instead.
 
-==== Debugging With VisualStudio Code
+=== Debugging With VisualStudio Code
```

Comment:
In my opinion, the important categories are Debugging Backend vs Debugging Frontend instead of Debugging with Visual Studio vs Debugging with Google Chrome Developer tools.

My proposal for categories is:
- Debugging Backend
- Debugging Frontend
  -- Google Chrome Developer Tools (IMO first because it is more complete)
  -- Visua Studio Code

Reply:
We don't have any text here to set up Visual Studio Code for frontend debugging - that section is only for setting up VSS for backend debugging.

Reply:
(if someone has VSS instructions on how to set it up for frontend debugging, let's add that, too! :)